packages/blockchain-wallet-v4-frontend/src/scenes/Signup/ProductPicker/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { connect, ConnectedProps } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import styled from 'styled-components'
+
+import { WalletOptionsType } from '@core/types'
+import { Icon, Image, Text } from 'blockchain-info-components'
+import { actions, selectors } from 'data'
+import { RootState } from 'data/rootReducer'
+
+const SceneWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+`
+
+const ContentWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin-top: 2rem;
+
+  > div {
+    cursor: pointer;
+
+    &:not(:last-child) {
+      margin-right: 24px;
+    }
+  }
+`
+
+const StyledDiv = styled.div``
+
+const ProductPicker: React.FunctionComponent<{
+  domains: {
+    comWalletApp: string
+    exchange: string
+  }
+  routerActions: { push: Function }
+  runGoals: Function
+  saveGoal: Function
+}> = (props) => {
+  const handleWalletLink = () => {
+    const { routerActions, runGoals, saveGoal } = props
+    routerActions.push(`${props.domains.comWalletApp}/home`)
+    // for first time login users we need to run goal since this is a first page we show them
+    saveGoal('welcomeModal', { firstLogin: true })
+    runGoals()
+  }
+
+  const handleExchangeLink = () => {
+    const { routerActions } = props
+    routerActions.push(`${props.domains.exchange}/trade`)
+  }
+
+  return (
+    <SceneWrapper>
+      <Icon color='success' name='checkmark-circle-filled' size='2rem' />
+      <Text size='1.25rem' weight={600} color='white' style={{ marginTop: '15px' }}>
+        <FormattedMessage
+          id='scenes.productPicker.account_created'
+          defaultMessage='Account Created!'
+        />
+      </Text>
+      <Text
```

Comment:
you might wrap this in a styled component and put all the styles there

Reply:
Na, I prefer to keep it as simple as possible.